newIDE/app/public/external/jfxr/jfxr-main.js

```diff
@@ -51,17 +51,26 @@ const saveSoundEffect = pathEditor => {
   });
 };
 
-// Wait for the window to be fully initialized before sending the
-// ready event. Don't use DOMContentLoaded as it was observed to be fired
-// even if jfxr DOM/scripts are not yet loaded.
+// Repeatedly try to gain access to jfxr's control element and its methods
```

Comment:
It's a "controller" in the AngularJS 1.x terms:

```
// Repeatedly try to gain access to jfxr controller.
// This is an "aggressive" approach needed because third party scripts like analytics
// can slow down the load process - so the Angular.js controller may not be initialized.
```